### ISSUE TITLE
About the weight decay

### ISSUE THREAD

**Issue by hungsing92** [Roll920/ThiNet#8]
Hi, 

想请问一下你训练时的weight_decay参数是多少? 我用weight_decay=0.0005 finetune做检测任务的时候, 网络很难收敛.

Best!

**Comment by Roll920** [comment]
@hungsing92 We set the batch size to 8 in training, and fine tune the model with 10^-3 learning rate for the first 120k iterations, then 10^-4 and 10^-5 for next two 30k iterations, respectively. We use SGD to optimize the training process where the weight decay is set to 0.0005, momentum is 0.9 and gamma is 0.1.
